utils.py

```diff
@@ -33,23 +33,21 @@ def get_reference_rates():
 
 
 def read_data(sub_dir, file_name):
-    # read list of deposits, sales, dividend payments and wire transfers
+    # read list of deposits, sales, dividend payments and currency conversions
     # sort them to ensure that they are in chronological order
     file_path = os.path.join(sub_dir, file_name)
-    df_deposits = pd.read_excel(
-        file_path, sheet_name="deposits", parse_dates=["date"]
-    ).sort_values("date")
-    df_sales = pd.read_excel(
-        file_path, sheet_name="sales", parse_dates=["date"]
-    ).sort_values("date")
-    df_dividends = pd.read_excel(
-        file_path, sheet_name="dividends", parse_dates=["date"]
-    ).sort_values("date")
-    df_wire_transfers = pd.read_excel(
-        file_path, sheet_name="wire_transfers", parse_dates=["date"]
-    ).sort_values("date")
-
-    return df_deposits, df_sales, df_dividends, df_wire_transfers
+    with pd.ExcelFile(file_path) as xls:
+        forex_sheet = "wire_transfers" if "wire_transfers" in xls.sheet_names else "currency conversion to EUR"
+        if forex_sheet == "wire_transfers":
+            print('"wire_transfers" as a sheet name is deprecated and discouraged; '
+                  'use "currency conversion to EUR" instead')
```

Comment:
nitpick
i usually prefer worksheet names intended for the use with pandas to not have spaces, this e.g. allows then to refer to these columns by `df.wire_transfers` instead of having to do `df["wire transfers"]`

could we shorten this to `forex_to_eur` for instance?

Reply:
though not too important, to let's just leave this as is